creator/events/migrations/0022_add_referral_token.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.24 on 2022-01-18 07:27
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('referral_tokens', '0003_add_referral_token'),
+        ('events', '0021_add_template_download'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='event',
+            name='referral_token',
+            field=models.ForeignKey(blank=True, help_text='Referral token related to this event', null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='events', to='referral_tokens.ReferralToken'),
+        ),
+        migrations.AlterField(
```

Comment:
This probably shouldn't be in the migration since you only made a change that adds the referral token fk to the event table